src/platform/datapath_winuser.c

```diff
@@ -2564,6 +2564,11 @@ CxPlatDataPathSocketContextShutdown(
 
     CxPlatRundownUninitialize(&SocketProc->UpcallRundown);
 
+    QuicTraceLogVerbose(
+        DatapathSocketContextComplete,
+        "[data][%p] Socket context shutdown",
```

Comment:
All the events you added, I don't think you should be logging the "parent". Since there are multiple child objects, you won't be able to distinguish which one.